### PR TITLE
Short hashes

### PR DESCRIPTION
**Pull request by Nancy-Badran**
Resolves StatCan/daaas#257

### CODE REVIEW

File: .github/workflows/build-cpu.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ name: build cpu
 on:
   push:
     branches:
+      - Short-Haches
```
Comment (sylus) [inline .github/workflows/build-cpu.yml:17]: Can we remove this branch, it should only happen on master? Assume just for testing?

Also is spelled wrong, should be hashes

Comment (Nancy-Badran) [reply .github/workflows/build-cpu.yml]: Yeah, I was testing the short sha ,

**Review by brendangadd — CHANGES_REQUESTED** [review]
Let's use eight characters.
* `git rev-parse --short` defaults to seven
* GitLab uses eight

**Comment by sylus** [comment]
I think this needs to be rebased from latest?

**Comment by Nancy-Badran** [comment]
Yeah @sylus , I will rebase it from latest, thanks

**Comment by Nancy-Badran** [comment]
workflow changed, proposed changes in PR are out of date

**Pull request closed without merging**